hack/update-all.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# A single script that runs a predefined set of update-* scripts, as they often go together.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+SILENT=true
+ALL=false
+
+declare -r color_red="\033[0;31m"
+declare -r color_yellow="\033[0;33m"
+declare -r color_norm="\033[0m"
+
+while getopts ":va" opt; do
+	case $opt in
+		a)
+			ALL=true
+			;;
+		v)
+			SILENT=false
+			;;
+		\?)
+			echo "Invalid flag: -$OPTARG" >&2
+			exit 1
+			;;
+	esac
+done
+
+trap 'exit 1' SIGINT
+
+if $SILENT ; then
+	echo "Running in the silent mode, run with -v if you want to see script logs."
+fi
+
+if ! $ALL ; then
+	echo "Running in short-circuit mode; run with -a to force all scripts to run."
+fi
+
+BASH_TARGETS="
+	generated-completions
+	generated-conversions
+	generated-deep-copies
+	generated-docs
+	generated-swagger-descriptions
+	generated-swagger-spec"
+
+
+for t in $BASH_TARGETS
+do
+	echo -e "${color_yellow}Updating $t${color_norm}"
+	if $SILENT ; then
+		if ! bash "$OS_ROOT/hack/update-$t.sh" 1> /dev/null; then
```

Comment:
Technically someone could put something other than Bash in their shebang, so we don't really need the `bash` her